ftw/testbrowser/core.py

```diff
@@ -206,7 +208,7 @@ def open_html(self, html):
         :type html: string or file-like object
         :returns: The browser object.
         """
-        self.response = self._load_html(html)
+        self.response = self._load_html(html, subparser=self.parser)
```

Comment:
I think we always want to use the custom subparser when parsing HTML but never want it when parsing XML, right?

How about wrapping `lxml.html.parse` and inserting the subparser there?
e.g.
```python
def parse_html(html):
    return lxml.html.parse(html, TestbrowserHTMLParser())
```

And then replace `lxml.html.parse` with `parse_html`.
I think you could revert most changes here.
I would prefer this approach.